src/js/utils/can-autoplay.js

```diff
@@ -25,21 +25,42 @@
+    }
+    return URL.createObjectURL(videoBlob);
+}
+
+// Catch errors caused by sites replacing the `URL` global with another value
+function getObjectURL() {
+    try {
+        return createObjectURL(window.URL);
+    } catch (e) {
+        console.error(window.URL, e);
```

Comment:
Do we want to log this?

Reply:
See the description above. I believe we do since site developers are making a big mistake. It would be even better for browsers to make `window.URL` read-only.